ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -5348,6 +5348,7 @@ policy-map type pbr PM_PBR_BREAKOUT
 ```eos
 !
 router bfd
+   slow-timer 5000
```

Comment:
Please add `slow-timer` entry  in `/j2templates/documentation/router-bfd.j2`.

Reply:
added bfd tuning section for it, like under router_bgp